lmtp_server_test.go

```diff
@@ -195,3 +197,94 @@ func TestServer_LMTP_DuplicatedRcpt(t *testing.T) {
 		t.Fatal("Invalid number of sent messages:", be.messages, be.anonmsgs)
 	}
 }
+
+func TestServer_LMTP_NetworkDefaultBehaviorUnix(t *testing.T) {
+	be := new(backend)
+	s := smtp.NewServer(be)
+	s.Domain = "localhost"
+	s.AllowInsecureAuth = true
+
+	s.Network = ""
+	s.LMTP = true
+	s.Addr = "./testsocket"
```

Comment:
We can't do that, this will clutter the current working directory.

Reply:
We could put it in os.TempDir()